tests/utils.py

```diff
@@ -94,6 +94,23 @@ def require_tf(test_case):
     return test_case
 
 
+def multigpu(test_case):
```

Comment:
`require_multigpu` for consistency?

Reply:
I agree, `require_multigpu` would be nice!

Reply:
done